src/dataconnect/freeTrial.ts

```diff
@@ -25,9 +59,9 @@ export function printFreeTrialUnavailable(
     `To use a different database in the same instance, ${clc.bold(`change the ${clc.blue("instanceId")} to "${instanceId}"`)} in ` +
     `${clc.green(configYamlPath)}. (Also, update the ${clc.blue("database")} field (i.e. DB name in the instance) ` +
     `and ${clc.blue("location")} as needed.)`;
-  utils.logLabeledBullet("dataconnect", reuseHint);
-  utils.logLabeledBullet(
-    "dataconnect",
+  utils.logLabeledError("data connect", reuseHint);
```

Comment:
Do we need all of these to be errors? The above can be seen as an error, but this one here and below are more like hints (informational). I don't have a strong preference here, but if you do decide to go with all errors, please also update the logs in the if-branch above

Reply:
Will do - I updated them for msotly visual reasons - it was a bit awkward having these semantically linked messages print out with different color badges.